lib/brainstem/concerns/controller_dsl.rb

```diff
@@ -71,40 +68,97 @@ def nodoc!
           configuration[brainstem_params_context][:nodoc] = true
         end
 
+        #
+        # Specifies which presenter is used for the controller / action.
+        # By default, expects presentation on all methods, and falls back to the
+        # class derived from +brainstem_model_name+ if a name is not
+        # given.
+        #
+        # Setting the +:nodoc+ option marks this presenter as 'internal use only',
+        # and causes formatters to display this as not indicated.
+        #
+        # @param [Class] target_class the target class of the presenter (i.e
+        #   the model it presents)
+        # @param [Hash] options options to record with the presenter
+        # @option [Boolean] options :nodoc whether this presenter should not
+        #   be output in the documentation.
+        #
+        #
+        def presents(target_class = :default, options = { nodoc: false })
+          raise "`presents` must be a class (in #{self.to_s})" \
+            unless target_class.is_a?(Class) || target_class == :default || target_class.nil?
+
+          target_class = brainstem_model_class if target_class == :default
+          configuration[brainstem_params_context][:presents] = \
+            options.merge(target_class: target_class)
+        end
 
         #
-        # Changes context to a specific action context. Allows specification
-        # of per-action configuration.
+        # Specifies a title to be used in the description of a class. Can also
+        # be used for method section titles.
         #
-        # Instead of using this method, it's advised simply to use +actions+
-        # with a single method name. While marked as private, since it is
-        # usually used within a +class_eval+ block thanks to
-        # +brainstem_params+, this has little effect.
+        # Setting the +:nodoc+ option marks this title as 'internal use only',
+        # and causes formatters to fall back to the controller constant or to
+        # the action name as appropriate. If you are trying to set the entire
+        # controller or action as nondocumentable, instead, use the discrete
```

Comment:
What does discrete mean in this context?

Reply:
There's nothing discrete about it, removing it